### PR TITLE
chore(content-releases): add tracking

### PR DESCRIPTION
**Pull request by Feranchz**
### What does it do?

Add tracking events to content-release events.

### How to test it?

1. Create a new release
2. You should see a new tracking request on the network tab

1. Publish a release
2. You should see a new tracking request on the network tab with the right counts

### CODE REVIEW

File: packages/core/content-releases/server/src/controllers/release.ts [hunk 1]
```diff
@@ -158,8 +158,27 @@ const releaseController = {
...
+        type: 'publish',
+      },
+    });
+
+    const countUnpublishActions = await releaseService.countActions({
+      filters: {
+        release: id,
+        type: 'unpublish',
+      },
+    });
```
Comment (remidej) [inline packages/core/content-releases/server/src/controllers/release.ts:173]: I think we can parallelize these queries

**Review by remidej — COMMENTED** [review]
Do we need to update the tracker to whitelist the new tracking events?

**Comment by Feranchz** [comment]
@remidej In order to avoid TS errors, yes, you need to add them

**Comment by Feranchz** [comment]
@remidej I guess we should, I will create a PR there as well

**Pull request merged**